src/anchor-navigation/styles.scss

```diff
@@ -117,7 +117,7 @@ $guide-line-offset: -2px; // Negative to expand 2px beyond the element
   margin-inline-start: awsui.$space-xs;
   @include styles.font-body-s;
   @include styles.font-smoothing;
-  font-weight: awsui.$font-button-weight;
+  font-weight: awsui.$font-weight-button;
```

Comment:
Especially now that `font-weight-button` is public/themable, I don't think it makes sense to use it here: it would be unexpected for changing the button weight to impact this "info" text. Suggest using `styles.$font-weight-bold;` instead

Reply:
("Updated" and "New" in this screenshot:)

![Screenshot 2024-11-19 at 09 20 56](https://github.com/user-attachments/assets/18a084a5-0f31-4d98-88a4-bb859e6995b1)

Reply:
Updated!